styles/_footer.scss

```diff
@@ -3,6 +3,8 @@
   padding: 1rem 2rem;
   margin: 2rem;
   min-width: 80vw;
+  flex-direction: row;
+  justify-content: center;
 
   padding: 1rem;
```

Comment:
There are two padding properties, we can remove the padding altogether as it's creating a width issue in mobile view.